src/query-commands.ts

```diff
@@ -116,15 +121,19 @@ const queryExplainDoc = {
 };
 const queryExplain = async (params: CmdParams) => {
     let query = params.input;
-    let result = await WAII.Query.describe({ query: query });
-    switch (params.opts['format']) {
+    if(query) {
+      let result = await WAII.Query.describe({query: query});
+      switch (params.opts['format']) {
         case 'json': {
-            console.log(JSON.stringify(result, null, 2));
-            break;
+          console.log(JSON.stringify(result, null, 2));
+          break;
         }
         default: {
-            log_query_explain_result(result);
+          log_query_explain_result(result);
         }
+      }
+    } else {
+      console.log('Query explain requires a query.')
```

Comment:
you should throw exception and log error instead of info.

Maybe we should have a common function to raise error for all other cases too.

Reply:
A better way to do it (it is a style comment, but can prevent issue too, which i think it is important) is. 

if (..some field is missing) {
    raise-exception ..
}

rest of the logic.

This can avoid wrap all logics in another `if`, and make sure we terminate the program when something bad happens.

Reply:
Can you check again.